client/src/panels/panel_declarations/finances/vote_stat/in_year_vote_stat_breakdown.js

```diff
@@ -106,7 +110,7 @@ const planned_vote_or_stat_render = (vs) =>
       name: "root",
       color: "white",
       children: _.map(data, (d, i) =>
-        _.extend(d, {
+        _.extend(_.clone(d), {
           id: i,
```

Comment:
More directly, could be 
```
_.map(data, (d, i) => ({
  ...d,
  id: i,
  // etc.
```

Intent's a bit clearer and avoids all the lodash util nesting (otherwise, would suggest chaining instead, at least within the `map`).